docs/install.md

```diff
@@ -26,30 +30,51 @@ CentOS 7:
     $ pip3 install --user vcd-cli
 ```
 
-## macOS
+## Photon OS 
 
-On macOS, open a Terminal and enter the commands listed below (skip those that refer to a component already installed on your mac):
+Photon OS minimal installs lack standard tools like pip3 and even ping,
+so you need to install a number of packages using tdnf.  There are also
+differences between Photon 1 and 2 distributions, so setup differs.
 
-Install `Xcode Command Line Tools`:
+Photon 1:
+``` shell
+    $ tdnf install -y gcc glibc-devel glibc-lang binutils python3-devel linux-api-headers
+    $ pip3 install --user vcd-cli
+```
 
+Photon 2:
 ``` shell
-    $ xcode-select --install
+    $ tdnf install -y build-essential python3-setuptools python3-tools python3-pip python3-devel
+    $ pip3 install --user vcd-cli
 ```
 
+## Mac OS X
+
+Open a Terminal and enter the commands listed below.  Skip those that
+refer to a component already installed on your Mac. 
+
+Install `Xcode Command Line Tools`:
+``` shell
+    $ xcode-select --install
+```
 Press `Install` and accept the license terms.
 
 Install `pip`:
-
 ``` shell
     $ sudo easy_install pip
```

Comment:
Is it not pip3?

Reply:
Just checked and it's 'pip'.  There is no pip3 installation package.  easy_install command installs both.